packages/core/src/external/athenahealth/index.ts

```diff
@@ -257,11 +271,23 @@ class AthenaHealthApi {
           })
           .catch(processAsyncError("Error saving to s3 @ AthenaHealth - getPatientViaSearch"));
       }
-      const searchSet = patientSearchResourceSchema.parse(response.data);
-      if (searchSet.entry.length > 1) {
-        throw new NotFoundError("More than one AthenaHealth patient found");
+      const searchSet = patientSearchResourceSchema.safeParse(response.data);
+      if (!searchSet.success) {
+        capture.message(`Patient search set could not be parsed @ AthenaHealth`, {
```

Comment:
ditto: maybe "Error parsing patient search response from Athena"?

and log :)

Reply:
Done.